app/src/components/features/rules/RulePairs/Pairs/HeadersRulePair/HeadersRulePairV2/HeadersPairModificationRowV2.js

```diff
@@ -60,7 +60,7 @@ const HeadersPairModificationRowV2 = ({
   );
```

Comment:
![image](https://github.com/requestly/requestly/assets/16779465/f85e2370-6f85-47a1-b927-e8bc1212d4a9)

No next in 3rd step of header rule

Reply:
the tours spotlight area does not respond well with width and height change. Example 👇

https://github.com/requestly/requestly/assets/53986600/3fa337da-4c25-41e9-b761-3956e37f89db

I have added an `autoMoveToNext` property here, this will make tour to move to next step as soon as user selects any one the modification options.